cms/models/titlemodels.py

```diff
@@ -32,7 +32,7 @@ class Title(models.Model):
                                   help_text=_("overwrite the title in the menu"))
     meta_description = models.TextField(_("description"), max_length=155, blank=True, null=True,
                                         help_text=_("The text displayed in search engines."))
-    slug = models.SlugField(_("slug"), max_length=255, db_index=True, unique=False)
+    slug = models.SlugField(_("slug"), max_length=255, db_index=True, unique=False, allow_unicode=True)
```

Comment:
This will fail on Django 1.8.

I can check for the django version and if it's higher than 1.8 then apply the `allow_unicode=True` unless leave the slug field untouched.

Any idea how to handle @czpython ?

Reply:
Django 1.8 is officially supported from Django untill april this year. I think that is a good idea to make it compatible with Django 1.8.

One approach can be:
```
try:
    models.SlugField(allow_unicode=True)
except TypeError:
    slug_extra_kwargs = {}
else:
    slug_extra_kwargs = {'allow_unicode': True}

...
slug = models.SlugField(_("slug"), max_length=255, db_index=True, unique=False, **slug_extra_kwargs)
```
Or something like that

Reply:
Hi All,
I tried to modified it in my project to allows unicode slugs.
cms/models/titlemodels.py
slug = models.SlugField(_("slug"), max_length=255, db_index=True, unique=False, allow_unicode=True)

it doesn't work, it complaint 'Slug must not be empty.'
django version:  1.11.26
Django-CMS version: 3.7 

any update for this issue?
thanks all